README.md

```diff
@@ -61,92 +58,30 @@ Once you've provided the profile, you can deploy the CloudFormation stack using
 
 IMPORTANT: when specifying the profile in your CloudFormation template, you must specify the Profile Name, NOT the Secret Name
 
-Right:
+Correct usage:
 ```
   "Profile" : "ProfileName"
 ```
-Wrong:
+Incorrect usage:
 ```
   "Profile" : "cfn/atlas/profile/ProfileName"
 ```
 
 # Logging 
 
-Logging for AWS CloudFormation Public extensions is currently disabled. AWS is evaluating if logging is useful for consumers of third party extensions, if this is something you need or would like to request please open a ticket directly with AWS Support. 
-
-# Testing the Provider
-
-Please see README for each resource for details on unit and integrated AWS testing.
-
-## Securing your Deployment 
-
-This repository contains utilities which use both the AWS and MongoDB Atlas APIs to progamatically create, destory, or otherwise manage resources in the AWS and MongoDB clouds. This can have costs or other impact so please use these tools with diligence and follow best practices. 
-
-The automated workflows require the following secrets to be placed into your Github forks' Settings/Secrets:
-
-```
-AWS_ACCESS_KEY_ID
-AWS_SECRET_ACCESS_KEY
-AWS_REGION
-ATLAS_PUBLIC_KEY
-ATLAS_PRIVATE_KEY
-ATLAS_ORG_ID
-```
-
-## Security - Setup 
-
-Step 1) Create and note your MongoDB Atlas API Key.
-Step 2) Create and note your AWS Access Key and AWS Secret Key ID.
-Step 3) Follow the Github docs on how to [create a Secret](https://docs.github.com/en/actions/configuring-and-managing-workflows/creating-and-storing-encrypted-secrets#creating-encrypted-secrets-for-a-repository) in your clone of this repository.
-
-## Requirements
-
-- [AWS CloudFormation CLI](https://github.com/aws-cloudformation/cloudformation-cli) 
-- (Optional - only need if building from source) [AWS CloudFormation CLI Go Plugin](https://github.com/aws-cloudformation/cloudformation-cli-go-plugin/) > v1.0
-- (Optional - only need if building from source) [Go](https://golang.org/doc/install) > v1.14 
-
+Logging for AWS CloudFormation Public extensions is currently disabled. AWS is evaluating if logging is useful for consumers of third party extensions, if this is something you need or would like to request please open a ticket directly with AWS Support.
 
-## Using the MongoDB Atlas CFN Resources 
+## Contributing
```

Comment:
FYI this shows as nested under logging
```suggestion
# Contributing
```
Ideally everything should be an h2 (##) but that may be a bigger change

Reply:
Yeah good point - updated across the docs being changed